package.json

```diff
@@ -78,6 +78,7 @@
     "@cowprotocol/cow-runner-game": "^0.2.9",
     "@cowprotocol/cow-sdk": "7.0.1-beta.0",
     "@cowprotocol/sdk-common": "^0.2.1-beta.0",
+    "@cowprotocol/sdk-composable": "^0.2.1-beta.0",
```

Comment:
Are there any changes related to composable cow? 
Or this is just because it was forgotten to be added in the first place?